### PR TITLE
Add & Remove tasks

### PR DESCRIPTION
**Pull request by kaizipaul**
Hello Reviewer! In this pull request the following changes were made:
- Created a modules folder to accommodate the JavaScript module code for the List object class I created.
- Added styling to accommodate the new functionalities.
- Used JavaScript to create a class so as to allow the creation and deletion of tasks dynamically on the DOM.
- Added functionality to the buttons for adding and removing tasks from the list using JavaScript.

**Thank You for reviewing my PR**

### CODE REVIEW

File: src/modules/listClass.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  static deleteItem = (index) => {
+    list.splice(index, 1);
+    list.forEach((e, i) => {
+      e.index = i + 1;
+    });
+    localStorage.setItem('listStorage', JSON.stringify(list));
+    Todolist.createList();
+  };
+}
```
Comment (chrystalme) [inline src/modules/listClass.js:62]: You have not implemented a function to edit the description of a Todo.

File: src/modules/listClass.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+export { list };
+export { addListItem };
+export { listContainer };
```
Comment (chrystalme) [inline src/modules/listClass.js]: You can do all your exports in one line. This makes your code cleaner and leaner. Making it more readable.

**Review by chrystalme — CHANGES_REQUESTED** [review]
## Hi @kaizipaul, 
You have done a good job so far!
However, there are some issues that you still need to work on to prepare your project for further evaluation
## Highlights 💡 
 - You have implemented the Add and Delete function in a concise and clever way. 👍🏽 
 - Linter working properly and passed ✔️ 
 - Beautiful design 🔆 
## Suggested changes ♻️ 
 - You have not implemented the Edit function of the Todo 
_Check the comments under the review._


You can also consider:
  - Using just one export for your exports

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA). Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @kaizipaul , 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ App works as expected👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**